gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,76 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
+        return cls.from_fits(hdu_list)
+
+    
+    @classmethod
+    def from_fits(cls, hdu, scheme=None):
+        """Read EnergyOffsetArray from a fits binary table.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the EnergyOffsetArray.
+        Returns
+        -------
+        cube : `~gammapy.background.EnergyOffsetArray`
+            EnergyOffsetArray object.
+        """
+
+        header = hdu[1].header
+        data = hdu[1].data
+        
+        # get offset and energy binning
+        offset_edges = _make_bin_edges_array(data['THETA_LO'], data['THETA_HI'])
+        energy_edges = _make_bin_edges_array(data['ENERG_LO'], data['ENERG_HI'])
+        
+        # get data
+        energy_offset_array=data['EnergyOffsetArray'][0]
+        return cls(offset_edges, energy_edges , energy_offset_array)
+
+    def to_table(self):
+        """Convert EnergyOffsetArray to astropy table format.
+
+        Returns
+        -------
+        table : `~astropy.table.Table`
+            Table containing the EnergyOffsetArray.
+        """
+        # table
+        table = Table()
+        table['THETA_LO']=np.expand_dims(self.offset[:-1],0)
```

Comment:
Since the column doesn't have the same shape, I can not create a Table if I am not using this np.expand...
I don't know how it can work in the method to_table() of the class Cube.py since x,y and energy have different dimension...
Maybe there is a cleaner way to do this